File: src/main/java/chapter01/primes/PrimeHelper.java
package chapter01.primes;

class PrimeHelper {

  static boolean isPrime(long num) {
    if (num == 0 || num == 1) {
      return false;
    } else {
      for (int i = 2; i * i <= num; i++) {
        if (num % i == 0) {
          return false;
        }
      }
      return true;
    }
  }
}

File: src/main/kotlin/chapter01/primes/PrimeHelper.kt
package chapter01.primes

object PrimeHelper {

    fun isPrime(num: Long): Boolean {
      if (num == 0L || num == 1L) {
        return false
      } else {
        var i = 2
        while (i * i <= num) {
          if (num % i == 0L) {
            return false
          }
          i++
        }
        return true
      }
    }
}